docs/components/dartpad/top-app-bar/contextual/dartpad_metadata.yaml

```diff
@@ -0,0 +1,4 @@
+name: Material Contextual Action Bar Demonstration
```

Comment:
Should this be App Bar not Action Bar?

Reply:
It isn't according to the dev docs 

https://github.com/JoseAlba/doc-material-components/blob/mingjane-doc-branch/top-app-bar/Flutter/Flutter-top-app-bar.md